llvm/include/llvm/IR/RepoTicket.h

```diff
@@ -59,46 +58,55 @@ struct GONumber {
 
 /// A structure of a global object (GO) information.
 struct GOInfo {
-  /// GO's initial hash value which does not include the hash of its dependents.
+  /// The Initial-digest is the hash of an object (a function or global
```

Comment:
I think this should use the same style for InitialDigest e.g.
"The **InitialDigest** is the hash of an object **itself** (a function of global variable) without consideration for any references to other objects."

Reply:
Fixed, thanks.